docs/vectorizer-quick-start-voyage.md

```diff
@@ -0,0 +1,130 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance. We also show how simple it is to do semantic search on the automatically embedded data!  
+We recommend to use voyage ai as an embedding provider, to follow this tutorial you need to have a voyage ai account and an api key. You can get one [here](https://voyageai.com/).
```

Comment:
I don't think that we need to provide a recommendation. They're already on this page.

```suggestion
To follow this tutorial you need to have a Voyage AI account API key. You can get one [here](https://voyageai.com/).
```